src/components/atoms/Plugin/IFrame/hooks.ts

```diff
@@ -228,3 +233,14 @@ function runScript(oldScript: HTMLScriptElement) {
     }
   });
 }
+
+function linksToHead(doc: Document, links: NodeListOf<HTMLLinkElement>) {
+  for (const link of links) {
+    const newLink = document.createElement("link");
+    newLink.rel = link.rel;
+    newLink.href = link.href;
```

Comment:
```suggestion
    for (const attr of newLink.attributes) {
      newLink.setAttribute(attr.name, attr.value);
    }
```

Reply:
do you mean `for (const attr of link.attributes)` ? Not newLink?

Reply:
Might be updated [here](https://github.com/reearth/reearth-web/pull/322/commits/b1879c0e926dfa34b636a872abfa7a078e4d8615)